### ISSUE TITLE
Webauthn support

### ISSUE THREAD

**Issue by ThraaxSession** [micronaut-projects/micronaut-security#931]

**Comment by sdelamo** [comment]
Some links: https://webauthn.guide/

**Comment by ThraaxSession** [comment]
@sdelamo yes I know this site. My question is if you plan to integrate the FIDO standard or if we need to integrate any Lib and mix micronaut code with it up.

**Comment by sdelamo** [comment]
@ThraaxSession We need to consider it first. No decision made yet whether to integrate it or note.

**Comment by ThraaxSession** [comment]
Hi @sdelamo when can you give feedback?

**Comment by baylorpaul** [comment]
I've created [a WebAuthn/Passkeys library for Micronaut](https://github.com/baylorpaul/webauthn4j-micronaut). Please check that out.

It also includes a sample Micronaut app, and a sample web UI.

I hope that helps get more developers using Passkeys. Passkeys seem like they'll be a great solution, but it took an inordinate amount of time and research to get that project pulled together. I can't imagine folks being able to casually add passkey support without libraries like the one I've made. It's based on [WebAuthn4J](https://github.com/webauthn4j/webauthn4j), which was instrumental, but it still didn't fill in all the pieces or answer a number of questions for a working app. I hope this bridges the gap for many developers.
